analyzers/src/Directory.Build.props

```diff
@@ -18,4 +20,8 @@
     </Compile>
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+  <Target Name="CleanBinaries" AfterTargets="Clean">
```

Comment:
_Rebase hint:_ This file was renamed to `.targets`
Most of the content was moved to `..\Analyzers.targets` and is shared between `src` and `tests`.
Your changes makes sense to remain in this `Directory.Build.targets`.